lib/ansible/modules/database/misc/elasticsearch_plugin.py

```diff
@@ -35,6 +35,11 @@
 version_added: "2.0"
 author: Mathew Davies (@ThePixelDeveloper)
 options:
+    additional_params:
```

Comment:
Fields that accept arbitrary settings like this are highly discouraged. From the Developer Guide:

> Modules should not require that a user know all the underlying options of an api/tool to be used. For instance, if the legal values for a required module parameter cannot be documented, that’s a sign that the module would be rejected.

This could probably be done as a `batch` parameter and an `xpack_security_enabled` parameter, both booleans.